### PR TITLE
Test integration

### PR DESCRIPTION
**Pull request by shimont**

### CODE REVIEW

File: test/integration/install.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+'use strict'
```
Comment (romanlab) [inline test/integration/install.test.js:1]: no need for strict mode

File: test/integration/norun.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict'
```
Comment (romanlab) [inline test/integration/norun.test.js:1]: no need for strict mode

File: test/integration/install.test.js (hunk 1 above)
Comment (shimont) [reply test/integration/install.test.js:1]: i was getting errors on node4

File: test/integration/norun.test.js (hunk 2 above)
Comment (shimont) [reply test/integration/norun.test.js:1]: i was getting errors on node4

**Pull request merged**